Xwt.XamMac/Xwt.Mac/OutlineViewBackend.cs

```diff
@@ -26,7 +26,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-global using ObjCRuntime;
+using ObjCRuntime;
```

Comment:
If you get rid of this, you're going to have to add this line to several other files.